packages/styleguide/src/components/Button/Button.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { oneOf } from 'prop-types';
+import styled from 'styled-components';
+import cx from 'classnames';
+
+import { rem } from './../../style/utils';
+
+const CLASS_ROOT = 'btn';
+
+const propTypes = {
+  variant: oneOf(['preview', 'code-toggle'])
+};
+
+/* Styles have to be on the top because of Tag and TagTitle */
+const StyledButton = styled.button`
+  display: inline-block;
+  min-height: ${rem(24)};
+  margin-bottom: ${props => props.theme.spaces.small};
+  padding: 0 ${props => props.theme.spaces.small};
+  vertical-align: middle;
+  text-align: center;
+  cursor: pointer;
+  appearance: none;
+  border: 0px solid transparent;
+  background: transparent;
+  font-family: ${props => props.theme.fontFamily};
+  font-weight: ${props => props.theme.fontWeights.bold};
+  font-size: ${props => props.theme.fontSizes.small};
+  color: ${props => props.theme.colors.black};
+
+  &--preview {
```

Comment:
Ja osobne by som sem tento preview variant nedaval, padding-right mi pride ze je to zmena, ktora je potrebna na konkretnom mieste. Uppercase by som vyhodil uplne, kvoli konzistencii.

Reply:
vyhodím ho